nemo_curator/datasets/doc_dataset.py

```diff
@@ -160,6 +160,7 @@ def to_json(
         output_path: str,
         write_to_filename: Union[bool, str] = False,
         keep_filename_column: bool = False,
+        partition_on: Optional[str] = None,
     ):
         """
         See nemo_curator.utils.distributed_utils.write_to_disk docstring for parameters.
```

Comment:
Right now we just tell the user to look at the `write_to_disk` docstring, but it would probably be better if we had a separate docstring here, too.

Reply:
Ageed, Added it here: https://github.com/NVIDIA/NeMo-Curator/pull/519/commits/d16047343f7413ee3f82df4fc4d38b032419e146